addons/event/models/event_mail.py

```diff
@@ -184,10 +184,10 @@ def execute(self):
                 # do not send emails if the mailing was scheduled before the event but the event is over
                 if scheduler.scheduled_date <= now and (scheduler.interval_type != 'before_event' or scheduler.event_id.date_end > now):
                     scheduler.event_id.mail_attendees(scheduler.template_ref.id)
-                    # Mail is sent to all attendees (unconfirmed as well), so count all attendees
+                    notified_registrations = scheduler.event_id.registration_ids.filtered(lambda r: r.state not in ('cancel', 'draft'))
                     scheduler.update({
                         'mail_done': True,
-                        'mail_count_done': scheduler.event_id.seats_expected,
+                        'mail_count_done': len(notified_registrations)
```

Comment:
At least in 17 isn't seats_expected being confirm + done ?

Reply:
Ah yes, in 17 (maybe 16 something) we changed that because it didn't count draft despite them being notified.

We could revert it now indeed.